hassio-google-drive-backup/backup/static/working.jinja2

```diff
@@ -203,23 +203,23 @@
                If you are affected this means that in October 2022 the add-on will be unable to authorize with Google Drive and your backups will not get uploaded.  
             </p>
             <p>
-              While annoying, this change is also somewhat expected.  Using using Google Cloud credentials for a local installation like this is not something Google 'intended' for us to do, it isn't an officially supported way of using their services. We can still make due though.  
+              While annoying, this change is also somewhat expected.  Using Google Cloud credentials for a local installation like this is not something Google 'intended' for us to do, it isn't an officially supported way of using their services. We can still make do though.  
```

Comment:
I had to look this one up.  Turns out I've been spelling "make do" wrong for over 30 years.